docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -38,27 +39,31 @@ A replication endpoint must exist before you create a replication rule. To creat
    `library/**`     | `library/hello-world`(Y)<br> `library/my/hello-world`(Y)
    `{library,goharbor}/**` | `library/hello-world`(Y)<br> `goharbor/harbor-core`(Y)<br> `google/hello-world`(N)
    `1.?`      | `1.0`(Y)<br> `1.01`(N)
-1. Use the **Destination Registry** drop-down menu to select from the configured replication endpoints. 
-1. Enter the name of the namespace in which to replicate resources in the **Destination namespace** text box.
+1. If you are creating a Push-based replication rule, use the **Destination Registry** drop-down menu to select from the configured replication endpoints.
+1. For **Destination Namespace**, enter the name of the namespace in which to replicate resources in the  text box. If you do not enter a namespace, resources are placed in the same namespace as in the source registry.
 
-   If you do not enter a namespace, resources are placed in the same namespace as in the source registry. 
-
-   ![Destination and namespaces](../../../img/replication-rule4.png)
-   
    **NOTE:** Because of major API changes in the v2.0 release to support [OCI](https://github.com/opencontainers/distribution-spec).
-   You **can not** replicate from harbor 1.x to 2.0, and you **can not** replicate artifacts with **manifest list** from 2.0 to 1.x. 
-   
+   You **can not** replicate from Harbor v1.x to v2.0 and later, and you **can not** replicate artifacts with **manifest list** from v2.0 and later to v1.x.
+
+1. Use the Destination Flattening drop-down to select how you want Harbor treat to image hierarchy when replicating images. Depending on what you select, Harbor will remove the same number of levels from the image's hierarchy, starting from the left, when replicating an image into your chosen destination namespace.
```

Comment:
Please add a `note` for this section:
If the replicating resources are Chartmuseum charts, the destination chart name hierarchy can only be 2 levels: namespace/chart, which means trying to put the chart under a repository named a/b/chart is invalid.
This will be added in the tooltips on UI as well

Reply:
thanks, saw your comment on https://github.com/goharbor/harbor/issues/15074. I added a note into the docs, does it make sense? Do we want to recommend that users also flatten all levels for rules that will replicate charts?

Reply:
As the `destination namespace` can contain multiple levels as well, `a/b/c` for example, even though flattening all levels, the resulting hierarchy `a/b/c/chart` doesn't meet the requirement either.

So how about don't suggest for setting which options and just clarify the resulting hierarchy should be in format `namespace/chart` ?

Reply:
Thanks for the explanation @ywk253100! i updated the wording here to align with you recommendation.